pkg/containerd/cmd/service/start.go

```diff
@@ -40,55 +41,65 @@ func start(args []string) {
 	}
 
 	service := args[0]
-	rootfs := filepath.Join("/containers/services", service, "rootfs")
+
 	log.Infof("Starting service: %q", service)
 	log := log.WithFields(log.Fields{
 		"service": service,
 	})
 
-	client, err := containerd.New(*sock)
+	id, pid, msg, err := start(service, *sock, *path, *dumpSpec)
+	if err != nil {
+		log.WithError(err).Fatal(msg)
+	}
+
+	log.Debugf("Started %s pid %d", id, pid)
+}
+
+func start(service, sock, path, dumpSpec string) (string, uint32, string, error) {
+	rootfs := filepath.Join(path, service, "rootfs")
+
+	client, err := containerd.New(sock)
 	if err != nil {
-		log.WithError(err).Fatal("creating containerd client")
+		return "", 0, "creating containerd client", err
```

Comment:
well I was trying to make the minimal change so as not to change the rest of the code that calls this, rather than refactoring everything. Personally I think `logrus` and `errors` and all these other libraries trying to improve the standard library are actually worse for almost all use cases, and wrapping errors is rarely useful...